quipucords/scanner/acs/runner.py

```diff
@@ -0,0 +1,86 @@
+"""abc scan task runner for acs scanner."""
```

Comment:
```suggestion
"""ABC scan task runner for ACS scanner."""
```
Consistent capitalization.

I've been meaning to comb through some of our messy docstrings that aren't properly capitalized in English sentence form. This wouldn't be the first, but let's not add more to the pile. 🙂